Sources/Request.swift

```diff
@@ -4,14 +4,17 @@ public struct Request: Message {
     public var version: Version
     public var headers: Headers
     public var body: Body
-    public var storage: [String: Any] = [:]
+    public var cookies: [String: String]
```

Comment:
There a number of other headers that are fundamental to HTTP and the cookie header in `Request` is not special like the cookie header in `Response` is. If we make `cookies` a stored property, why not make `contentLength` a stored property? or `contentType`? They're much more important than cookies. I think they should all be computed properties that use the `headers` property as storage. There's no need for another stored property if `headers` can store all headers with no problem. Having another stored property will be an overhead in the HTTP parser, cause it will have to check every header it parses to see if it's a cookie header and if so it stores in `cookies` instead of `headers`. If we add even more stored properties the overhead and complexity of the parser will increase even more. Because it's not good to have a lot of stored properties for headers and because cookies aren't special in `Request`, I think it's best to store cookies in `headers` and not have a `cookies` property on `Request`.